### ISSUE TITLE
Allow missing and/or empty files

### ISSUE THREAD

**Issue by reece** [EndBug/label-sync#263]
For a small organization, I'd like to have a common set of standardized labels and the option for some repos to add new labels. The config stanza below works well:

```
          config-file: |
            https://raw.githubusercontent.com/biocommons/.github/main/.github/labels.yml
            .github/labels.yml

```

For this to work, .github/labels.yml must exist and must not be empty. If no new labels are intended, the file may contain just `[]`. 

At a minimum, this issue may help others achieve a similar outcome. 

In addition, please consider allowing a file to empty, in which case it's interpreted as `[]`.

**Comment by EndBug** [comment]
Good idea! I could add an opt-in input that treats 404s and missing files like an empty array :)
